tests/test_storage.py

```diff
@@ -29,7 +28,7 @@ def callback(*args, **kwargs):
 
 @pytest.mark.parametrize("adata_format", ["h5ad", "zarr"])
```

Comment:
Why don't we adopt the `.zrad` ending that we agreed on, here? 🤔

Reply:
It is ".zarr" to test also that even in this case the function understand that this is actually anndata.

Reply:
Make sense!